ui/js/controllers/jobs.js

```diff
@@ -129,14 +129,14 @@ treeherderApp.controller('ResultSetCtrl', [
          */
         $scope.pinAllShownJobs = function () {
             if (!thPinboard.spaceRemaining()) {
-                thNotify.send("Pinboard is full.  Can not pin any more jobs.",
+                thNotify.send(pinboardError,
                               "danger");
```

Comment:
L133 can be moved up one line now that the line length is reduced - that was the reason it was previously wrapped. Treeherder files vary, but they generally shoot for a wrap at either 80 or 100 chars, if I recall correctly. Mostly I look and generally follow the wrap limit apparent in the specific file.

Reply:
agreed  :)